MAUI/Popup/layout-customizations.md

```diff
@@ -658,3 +658,17 @@ public partial class MainPage : ContentPage
 {% endhighlight %}
 {% endtabs %}
 
+## How to show overlay background always in .NET MAUI Popup?

```

Comment:
default is true. please make the use case as false case.

Reply:
Addressed the concerns provided.